src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/utils/AsylumCaseUtils.java

```diff
@@ -30,6 +30,12 @@ public static boolean isAcceleratedDetainedAppeal(AsylumCase asylumCase) {
             .equals(YES);
     }
 
+    public static boolean isDetainedAppeal(AsylumCase asylumCase) {
```

Comment:
I think we have already this check line 23 in the same class **isAppellantInDetention(AsylumCase asylumCase) {**

Reply:
Good spot, removed it.